### ISSUE TITLE
Accessibility API access sometimes doesn't work unless using an alias

### ISSUE THREAD

**Issue by luckman212** [keycastr/keycastr#146]
I just downloaded 0.9.7 on a mac mini running 10.15
when I launch it, I get the dialog requesting Accessibility API access
I removed & re-added Keycastr as suggested but the process just repeats
Anyone else having this problem?

```
Could not create keyDown event tap!
Please grant KeyCastr access to the Accessibility API.
If KeyCastr already has access, please remove it and add it again.
```

![image](https://user-images.githubusercontent.com/1992842/67163384-26706c00-f33c-11e9-92d5-655be05f25d2.png)

![image](https://user-images.githubusercontent.com/1992842/67163393-4a33b200-f33c-11e9-9d42-36e104b6019d.png)

**Comment by NylonDiamond** [comment]
Same issue for me as well. I just moved the app to the Documents folder and created an alias to the Applications folder. It works for me for now.

**Comment by transducer** [comment]
The suggestion of @NylonDiamond works. But why? :)

**Comment by akitchen** [comment]
Unsure... It's expected to need to re-grant accessibility API permissions after an update or when the app settings change. This needs to be done by removing the app from the list and re-adding it, not simply re-checking the box.

I haven't been able to test on 10.15 to see if things are significantly different in a way we need to work around. I do know that notarization is increasingly important in macOS Catalina and beyond.

Managing security permissions is an area of KeyCastr I'm keep to improve, and I've been working on code signing and notarization to see if it makes any difference especially for Catalina. But since I'm not able to upgrade yet, I'm limited in being able to test it.

Closing this for now. Next release should be notarized and it will be good to know whether it helps.

**Comment by akitchen** [comment]
I got the app code signed and notarized, and I still see the same issue. And strangely, using an alias seems to work. I'm still on 10.14, so it's not specific to Catalina.

**Comment by diegohaz** [comment]
I'm also having this issue (weirdly got here exactly when @akitchen posted an update lol). How can I create an alias?

**Comment by akitchen** [comment]
This problem seemed to fix itself for me after a while, possibly after a restart. I can now use KeyCastr without an alias from its normal place in /Applications

**Comment by diegohaz** [comment]
It worked for me after enabling Input Monitoring:
![image](https://user-images.githubusercontent.com/3068563/67643517-001f7300-f8de-11e9-9dff-34d768f1a397.png)

**Comment by WShiBin** [comment]
@diegohaz thx

**Comment by gziolo** [comment]
@diegohaz, thanks it solved my issue as well 👍

**Comment by BirkhoffLee** [comment]
Rebooting my macOS solved the issue

**Comment by mfekadu** [comment]
@diegohaz thanks for the tip! It really helped! I hope the README update will help even more people 😄

**Comment by elia** [comment]
Should be fixed by #158, kudos to @mfekadu for putting together the instructions 👍
